setup.py

```diff
@@ -105,6 +107,14 @@ def has_ext_modules(self):
   sys.argv.remove('--nightly')
   sys.argv.pop(nightly_idx)
 
+if '--preview' in sys.argv:
+  preview_idx = sys.argv.index('--preview')
+  version = version + ".dev" + sys.argv[preview_idx + 1]
+  package = 'tf-nightly-2.0-preview'
+  project = 'tensorflow-io-preview'
```

Comment:
Should we rename this to be `tensorflow-io-2.0-preview` to be consistent with [tensorflow-estimator-2.0-preview](https://pypi.org/project/tensorflow-estimator-2.0-preview/)?

Reply:
@terrytangyuan Updated and changed to `tensorflow-io-2.0-preview`.